manager_rethink.go

```diff
@@ -62,6 +62,24 @@ func rdbFromPolicy(p Policy) (*rdbSchema, error) {
 	}, err
 }
 
+// NewRethinkManager initializes a new RethinkManager for given session, table name defaults
+// to "policies", policies can default to nil.
+func NewRethinkManager(session *r.Session, table string, policies map[string]Policy) *RethinkManager {
```

Comment:
I think consistency is more important here, let's not return an error (same to what the SQL factory does)